exporters/postgres/sql_servers_example.yml

```diff
@@ -24,6 +24,8 @@ jobs:
     enable_ping: true
     static_configs:
       - targets:
-          # " same as URL in cht-instances.yml      ": 'postgres://USERNAME:PASSWORD@DB_SERVER_IP/DATABASE
-          "172-17-0-1.local-ip.medicmobile.org:10464": 'postgres://cht_couch2pg:cht_couch2pg_password@172.17.0.1:5432/cht?sslmode=disable' # //NOSONAR - password is safe to commit
+          # change USERNAME, PASSWORD, DB_SERVER as needed. Likely DATABASE and PORT don't need to change.
+          # be sure each new server gets a unique name (eg "db1", "db2" etc.)
+          #     'postgres://USERNAME:PASSWORD@DB_SERVER_IP/DATABASE:PORT
+          "db1": 'postgres://cht_couch2pg:cht_couch2pg_password@172.17.0.1:5432/cht?sslmode=disable' # //NOSONAR - password is safe to commit
```

Comment:
So, it looks like this `db1` value gets pulling into Prometheus as the `target` label value for the metrics.  With that in mind, I think it might be useful to encourage folks to use the identifier of their DB server here (e.g. `DB_SERVER_IP`).  Not going to provide any functional value at this moment, but it is semantically significant and adds a detail to the metric that was not being captured before...

Reply:
yes - we have some data that might be helpful, don't hide it behind obtuse names! fixed in 69d2e57